sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,9 +1,23 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-10-08)
 
+### Breaking Changes
+
+- Throw Azure::Http::TransportException if creating new connection fails.
+- Response objects store Nullable<T>.
```

Comment:
Angle brackets don't render in markdown
```suggestion
- Response objects store Nullable\<T\>.
```
![image](https://user-images.githubusercontent.com/6527137/95532656-d730aa80-0995-11eb-9084-bf7489ae886e.png)

Reply:
Nice catch, fixed